docs/edgeql/funcops/string.rst

```diff
@@ -241,10 +241,38 @@ String
 ----------
 
 
+.. eql:function:: std::str_pad_start(string: str, n: int64, fill: str = ' ') \
+                    -> str
+
+    Return the input *string* padded at the start to the length *n*.
+
+    If the *string* is longer than *n*, then it is truncated to the
+    first *n* characters. Otherwise, the *string* is padded on the
+    left up to the total length *n* using *fill* characters (space by
+    default).
+
+    .. code-block:: edgeql-repl
+
+        db> SELECT str_pad_start('short', 10);
+        {'     short'}
+        db> SELECT str_pad_start('much too long', 10);
+        {'much too l'}
+        db> SELECT str_pad_start('short', 10, '.:');
+        {'.:.:.short'}
+
+
+----------
+
+
 .. eql:function:: std::str_lpad(string: str, n: int64, fill: str = ' ') -> str
```

Comment:
I'd add a separate page collecting all deprecated functions.

Reply:
OK, I've made a separate page for that.